src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
+<CollapserGroup>
+  <Collapser
+    id="select-your-query"
+    title="Select your query"
+  >
+  First, we'll go to the chart labeled **Web transactions time** and click **create an alert condition**. 
 
-Ideas for next steps: 
-
-* If you're confused about the above procedure, keep reading for some tips on creating a NRQL alert condition.
-* Read about [some recommended APM alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations).
-
-## Create a NRQL alert condition [#nrql-alert] 
-
-You can write your own [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) and use that as the basis for a NRQL alert condition. The following will walk you through how to create a condition. 
+<img
+  title="Alert condition from a chart"
+  alt="A screenshot depicting how to create an alert condition from a chart"
+  src={alertsCreateanAlertConditionFromaChart}
+/>
 
-### Step 1: Write your query [#step1-nrql]
+<img
+  title="Select your query"
+  alt="A screenshot demonstrating how to choose your query"
+  src={alertsSelectYourQuery}
+/>
+</Collapser>
 
-You can use a NRQL query to return data about how your environment is performing. Then, you can create a condition from that query.
+  <Collapser
+    id="name-your-condition"
+    title="Name your condition"
+  >
+    It's important to give your alert condition a specific name. Let's say you name this condition **response time** and then you create another condition for a completely different application and label that condition **response time** as well, then you won't be able to distinguish which condition is for which application. So, make sure to give your condition a specific and unique name. In this case, we'd name this condition **Response time: Example-app**.
 
-In this example, a condition is created for each host's CPU utilization (query), which will trigger a notification when the CPU utilization is above the 0.7% (threshold). Generally, you won't use thresholds below 1%, but this example simply shows you how to set up a threshold.
+  </Collapser>
 
-Find the [NRQL query builder](/docs/query-your-data/explore-query-data/query-builder/use-advanced-nrql-mode-query-data/) in the [all capabilities](https://one.newrelic.com/all-capabilities) section of the UI.
+  <Collapser
+    id="define-your-signal"
+    title="Define your signal"
+  >
+    Once you've named your condition, then you can make changes to the NRQL query if you'd like. For your first alert, we recommend not adjusting the NRQL query and leaving this section as is but if you'd like to learn more about how to use NRQL to customize your queries, visit our [docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/). 
```

Comment:
I think the "if you'd like" can probably be cut here. I THINK there should also be a comma after "as is" in the second sentence, but I'm not 100% sure.